backend/capellacollab/models/crud.py

```diff
@@ -0,0 +1,75 @@
+# Standard library:
+import typing as t
+
+# 3rd party:
+from sqlalchemy.orm import Session
+
+# 1st party:
+import capellacollab.projects.crud as projects_crud
+from capellacollab.models.models import EmptyModel, Model, NewModel, ToolDetails
+from capellacollab.projects.models import DatabaseProject
+from capellacollab.tools.models import Tool, Type, Version
+
+
+def get_all(db: Session, project_slug: str) -> t.List[Model]:
```

Comment:
Is it really worth it? This would create some redundancy with the name of the crud module. For exemple if `models.crud` is imported as `models_crud`, The function will be called with `models_crud.get_all_models`.

Reply:
It's not super important, but I think it's better to make the repository a bit more consistent and maybe `get_all_git_models` is even better.

Reply:
I renamed the crud functions with more explicit names. However I don’t intend yet to specify “git”, because these functions should not be specialised in one source type.